### PR TITLE
Conda environments

### PR DESCRIPTION
**Pull request by NoraLoose**
This PR does two small things:
* Adds the packages necessary for testing (`pre-commit`, `pytest`) to the two provided conda environments
* Updates the installation instructions by making the user aware that there are two conda environments to choose from

### CODE REVIEW

**Comment by dafyddstephenson** [comment]
looks good, would suggest `laptop or an HPC machine` --> `supported HPC system (environment management by Linux Environment Modules) or generic machine (environment managed by conda)`

File: README.md [hunk 1]
```diff
@@ -11,10 +11,15 @@ git clone https://github.com/CWorthy-ocean/C-Star.git
 cd C-Star
 ``` 
 
-Next, install and activate the following conda environment:
+Next, install **one** of the following conda environments, depending on whether you are working on a laptop or an HPC machine:
```
Comment (dafyddstephenson) [inline README.md]:
```
Next, install **one** of the following conda environments, depending on whether you are working on a supported HPC system (environment managed by Linux Environment Modules) or generic machine (environment managed by conda)"
```

**Comment by NoraLoose** [comment]
Like that?

File: docs/contributing.md [hunk 2]
```diff
@@ -2,13 +2,24 @@
 
 ## Conda environment
 
-You can install and activate the following conda environment:
+Install **one** of the following conda environments, depending on whether you are working on a laptop or an HPC machine:
```
Comment (dafyddstephenson) [inline docs/contributing.md]:
```
Next, install **one** of the following conda environments, depending on whether you are working on a supported HPC system (environment managed by Linux Environment Modules) or generic machine (environment managed by conda)"
```

File: docs/installation.md [hunk 3]
```diff
@@ -11,10 +11,15 @@ git clone https://github.com/CWorthy-ocean/C-Star.git
 cd C-Star
 ``` 
 
-Next, install and activate the following conda environment:
+Next, install **one** of the following conda environments, depending on whether you are working on a laptop or an HPC machine:
```
Comment (dafyddstephenson) [inline docs/installation.md]:
```
Next, install **one** of the following conda environments, depending on whether you are working on a supported HPC system (environment managed by Linux Environment Modules) or generic machine (environment managed by conda)"
```

File: docs/installation.md [hunk 4]
```diff
@@ -11,10 +11,15 @@ git clone https://github.com/CWorthy-ocean/C-Star.git
...
 
 ```
-conda env create -f ci/environment.yml
+# conda env create -f ci/environment.yml  # conda environment for laptop
```
Comment (dafyddstephenson) [inline docs/installation.md]: ``` conda env create -f ci/environment.yml  # conda environment for a generic machine```

**Review by dafyddstephenson — CHANGES_REQUESTED** [review]

**Comment by dafyddstephenson** [comment]
you beat me to it!

**Review by dafyddstephenson — APPROVED** [review]

**Pull request merged**